src/ResourceManager/Resources/Commands.Resources/help/New-AzureRmADAppCredential.md

```diff
@@ -15,7 +15,7 @@ Adds a credential to an existing application.
 
 ### ApplicationObjectIdWithPasswordParameterSet (Default)
 ```
-New-AzureRmADAppCredential -ObjectId <String> -Password <String> [-StartDate <DateTime>] [-EndDate <DateTime>]
+New-AzureRmADAppCredential -ObjectId <String> -Password <SecureString> [-StartDate <DateTime>] [-EndDate <DateTime>]
```

Comment:
Can you update the example to include using securestring?  This applies for all other files too (just add a command to create a new securestring: ConvertTo-SecureString with the -PlainText and -Force parameters added, and then use this secure string in place of the existing plain-text string).